app/models/user.rb

```diff
@@ -5,4 +5,9 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable
   has_many :rentals
   has_many :items, through: :rentals
+
+  validates :first_name, :last_name, :username, :adress, :encrypted_password, :email, presence: true
+  validates :first_name, uniqueness: { scope: :last_name }
```

Comment:
first name can't be unique so I would remove this?

Reply:
It means that the combination of that first and last name have to be unique, so noone can have the exact same first and last name, but yes, that might not be necessary.

Reply:
i see, smart. up to you :)